ledger/src/sigverify_shreds.rs

```diff
@@ -225,13 +225,13 @@ pub fn verify_shreds_gpu(
     out.set_pinnable();
     elems.push(perf_libs::Elems {
         #[allow(clippy::cast_ptr_alignment)]
```

Comment:
I wonder if `#[allow(clippy::cast_ptr_alignment)]` is needed

Reply:
Ah yes, nice catch. The other PR removed these `allow`s, so it's definitely ok. Adding it here